### PR TITLE
feat: Add TodayTile badge

### PR DESCRIPTION
**Pull request by bruce-glazier**
## Changes

  - Add Badge to TodayTile to display assigned surveys or consents that require action

## Screenshots

image

### CODE REVIEW

File: src/hooks/todayTile/useTodayTasks.tsx [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  const { activeSubjectId, activeProject } = useActiveProject();
+  const { accountHeaders } = useActiveAccount();
+
+  return useRestQuery(
+    'GET /v1/survey/projects/:projectId/responses',
+    {
+      projectId: activeProject!.id,
+      author: activeSubjectId!,
+      patientId: activeSubjectId!,
+      includeSurveyName: true,
```
Comment (sternetj) [inline src/hooks/todayTile/useTodayTasks.tsx:43]: Do we need the survey name?

File: src/hooks/todayTile/useTodayTasks.tsx [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    consentTasks?.filter((task) => task.status === 'proposed') ?? [];
+  const inProgressSurveyTasks =
+    surveyTasks?.filter((task) => task.status === 'in-progress') ?? [];
+  const newTasks = [
+    ...inProgressConsentTasks,
+    ...inProgressSurveyTasks,
+  ] as TodayTask[];
+
+  return {
+    doneTasks,
```
Comment (sternetj) [inline src/hooks/todayTile/useTodayTasks.tsx:116]: Is this used?

**Review by sternetj — DISMISSED** [review]

File: src/hooks/todayTile/useTodayTasks.tsx (hunk 1 above)
Comment (bruce-glazier) [reply src/hooks/todayTile/useTodayTasks.tsx]: No, we don't actually need this to display the badge, I've updated this to false.

File: src/hooks/todayTile/useTodayTasks.tsx (hunk 2 above)
Comment (bruce-glazier) [reply src/hooks/todayTile/useTodayTasks.tsx]: No, we don't have a use for this yet, I've remove it.

File: src/hooks/todayTile/useTodayTasks.tsx [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+  const { accountHeaders } = useActiveAccount();
+
+  return useRestQuery(
+    'GET /v1/survey/projects/:projectId/responses',
+    {
+      projectId: activeProject!.id,
+      author: activeSubjectId!,
+      patientId: activeSubjectId!,
+      includeSurveyName: false,
+      status: 'in-progress,complete',
```
Comment (sternetj) [inline src/hooks/todayTile/useTodayTasks.tsx:44]: We don't need to fetch `complete` anymore.

**Review by sternetj — APPROVED** [review]

**Review by namick — APPROVED** [review]

**Pull request merged**